frontend/src/features/map/overlays/missions/MissionCard.tsx

```diff
@@ -5,12 +5,24 @@ import styled from 'styled-components'
 import { editMissionInLocalStore } from '../../../../domain/use_cases/missions/editMissionInLocalStore'
 import { clearSelectedMissionOnMap } from '../../../../domain/use_cases/missions/selectMissionOnMap'
 import { useAppDispatch } from '../../../../hooks/useAppDispatch'
+import { useAppSelector } from '../../../../hooks/useAppSelector'
 import { MissionSourceTag } from '../../../../ui/MissionSourceTag'
 import { MissionStatusLabel } from '../../../../ui/MissionStatusLabel'
 import { missionTypesToString } from '../../../../utils/missionTypes'
 
-export function MissionCard({ feature, selected = false }: { feature: any; selected?: boolean }) {
+type MissionCardProps = {
+  feature: any
+  isOnlyHoverable?: boolean
+  selected?: boolean
+}
+export function MissionCard({ feature, isOnlyHoverable = false, selected = false }: MissionCardProps) {
   const dispatch = useAppDispatch()
+  const {
+    global: { displayMissionsLayer }
+  } = useAppSelector(state => state)
```

Comment:
Je pensais que useSelector(state=>state) générait un warning. En effet le composant va faire un rerender à chaque fois qu'il y a une modif dans redux.

```suggestion
  const displayMissionsLayer  = useAppSelector(state => state.global.displayMissionsLayer)
```